src/app/assets/javascripts/angular/app.js

```diff
@@ -8,26 +8,58 @@
   app.config(['$routeProvider', function($routeProvider){
     $routeProvider
       .when('/', {
-        templateUrl: 'ng-index.html',
-        controller: 'MainController'
+        templateUrl: 'location.html',
```

Comment:
default template for `'/'` is `location.html`. If user is not logged in, redirect to `'/login'`